modAionImpl/src/org/aion/zero/impl/sync/SyncMgr.java

```diff
@@ -393,11 +439,32 @@ public long getNetworkBestBlockNumber() {
     }
 
     public synchronized void shutdown() {
+        if (p2pLog.isDebugEnabled()) {
```

Comment:
minor: the check can be removed

Reply:
The check here is actually an optimization to skip the other if statements when debug is disabled. Would you still want it removed?

Reply:
I see, because you call the method in the log argument. Just ignore my comment.